apps/govern/common-util/constants/urls.ts

```diff
@@ -1 +1,3 @@
 export const GET_VEOLAS_URL = 'https://member.olas.network/';
+export const REPO_URL =
+  'https://github.com/valory-xyz/autonolas-frontend-mono/tree/main/apps/govern';
```

Comment:
the file structure here is pretty clear, not sure what this would achieve

Reply:
agree (with Josh). actually, "utils" sounds to me more like helper functions, I think even hooks are not included - should be separated. but this is just what I'm used to

Reply:
good point about the app urls (but not the repo urls maybe, at least I don't see where except in its own app it can be used)
but we can keep them in the `libs/util-constants/src/lib/urls` or `libs/util-constants/src/lib/appUrls`? wdyt